www.foia.gov/_layouts/home.html

```diff
@@ -0,0 +1,67 @@
+---
+---
+<!DOCTYPE html>
+<html>
+{% include head.html %}
+<body>
+{% include header.html %}
+<section class="usa-hero">
+  <div class="usa-grid">
+    <div class="usa-hero-callout usa-section-dark">
+      <h2><span class="usa-hero-callout-alt">{{ page.hero.hed }}</span></h2>
+      <p>{{ page.hero.text }}</p>
+      <div class="hero-unit_button-group">
+        <a class="usa-button usa-button-outline" href="#">Start your request</a>
+        <span>or</span>
+        <a class="usa-button usa-button-outline" href="#">Learn more about FOIA</a>
+      </div>
+    </div>
+  </div>
+</section>
+<section class="usa-grid three-column-summary">
+  <h2>{{ page.section_one.hed }}</h2>
+  <p class="secondary-text">If you know you’re ready to submit a request, <a href="#agency-search">continue below</a></p>
```

Comment:
Ending period?

Reply:
The mocks show a down arrow here that kind of takes the place of a period
<img width="440" alt="screen shot 2017-10-26 at 8 27 20 pm" src="https://user-images.githubusercontent.com/509309/32085407-18e3adbc-ba8c-11e7-814a-3149a1c0eed9.png">